### ISSUE TITLE
No such file or directory: 'data/vecs.lc.over100freq.txt.gz'

### ISSUE THREAD

**Issue by Chevalier1024** [heikeadel/global_normalization#1]
Could you please share the link of word2vec you use in your model? thanks

**Comment by Prarthana25** [comment]
Can you let me know the link for word2vec?

**Comment by Chevalier1024** [comment]
haha，thank you for your reply。I mean the word2vec file you use in your model. I found it in other page.I have another question. You split the sentence into six parts: left of e_i, e_i, right of e_i, left of e_j, e_j, right of e_j. Of course, you can do this in your training set because you have entities label. I don't know why you do it in test set. As for me ,test set hasn't entities label. Look forward to your reply.
